### PR TITLE
Signals reservation page #842199259802786

### PR DESCRIPTION
**Pull request by jessehon**
https://app.asana.com/0/683768070860536/842199259802786

### CODE REVIEW

File: app/javascript/App/serverStartup.tsx [hunk 1]
```diff
@@ -7,9 +7,25 @@ import createServerAppHash from './createServerAppHash'
...
 import createServerComponentHash from './createServerComponentHash'
 import CoinShow from './components/CoinShow'
+import SignalExamplePanel from './bundles/signalsPage/SignalExamplePanel'
+import SignalFaqPanel from './bundles/signalsPage/SignalFaqPanel'
+import SignalPopoverText from './bundles/signalsPage/SignalPopoverText'
+import SignalTeamMember from './bundles/signalsPage/SignalTeamMember'
 
 // Register the components so they can be rendered from Rails
-(ReactOnRails as any).register({
+;(ReactOnRails as any).register({
```
[the remark this answers is not in the document]
Comment (jessehon) [reply app/javascript/App/serverStartup.tsx]: I've removed it a few times in previous commits. Prettier keeps adding it back =(

[the remark this answers is not in the document]
Comment (hanchang) [reply app/javascript/App/serverStartup.tsx]: Weird, okay no worries, was just curious if there was some new syntactic magic I wasn't aware of, I'm fine to keep it in to appease Prettier.

File: app/controllers/kyber_controller.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+class KyberController < ApplicationController
+  def widget_callback
+    byebug
```
Comment (jessehon) [inline app/controllers/kyber_controller.rb]: Do we still want to remove this?

Comment (hanchang) [reply app/controllers/kyber_controller.rb]: This shouldn't even be in here...

**Pull request merged**